### PR TITLE
Revert "Bump puppeteer from 19.6.2 to 19.6.3 in /webdriver"

### PR DESCRIPTION
**Pull request by KyleJu**
Reverts web-platform-tests/wpt.fyi#3147

### CODE REVIEW

**Review by DanielRyanSmith — APPROVED** [review]

File: webapp/components/test/test-run.html [hunk 1]
```diff
@@ -60,7 +60,7 @@
     });
 
     test('timeFormat(iso string)', () => {
-      assert.equal(trf.timeFormat('2018-01-12T12:00:00Z'), 'Jan 12 at 12:00 PM');
+      assert.equal(trf.timeFormat('2018-01-12T12:00:00Z'), 'Jan 12 at 12:00 PM');
```
Comment (jcscottiii) [inline webapp/components/test/test-run.html:63]: Could you try to revert this line and let it run?

Comment (KyleJu) [reply webapp/components/test/test-run.html]: Good idea!

Comment (KyleJu) [reply webapp/components/test/test-run.html]: Work like a charm

Comment (DanielRyanSmith) [reply webapp/components/test/test-run.html]: So definitely a puppeteer issue

**Comment by KyleJu** [comment]
Filed  #3152

**Pull request merged**